recipe/meta.yaml

```
@@ -6,36 +6,52 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
-  sha256: 97e63b5603e085c6a56a12a95740010e75d9134e0aab767e0978b53fd8824f13
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+    sha256: 97e63b5603e085c6a56a12a95740010e75d9134e0aab767e0978b53fd8824f13
+  # file needed for tests, not included in the pypi tarball
+  - url: https://raw.githubusercontent.com/jacksonllee/iso639/v{{ version }}/CHANGELOG.md
+    sha256: 8f8da6ba0cce1e1e2065a1082c74f0e5d7144d18de050efd13552fa552d53746
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.8
+    - python
     - setuptools >=65.3.0
     - wheel
     - pip
   run:
-    - python >=3.8
+    - python
 
 test:
   imports:
     - iso639
-  commands:
-    - pip check
   requires:
     - pip
+    - pytest
+  source_files:
+    - tests
+    - CHANGELOG.md
+  commands:
+    - pip check
+    # see section tool.pytest.ini_options in pyproject.toml for the
+    # additional parameters
+    - pytest -vv --durations=0 --strict-markers tests
```

```suggestion
    - pytest -v tests
```
One `-v` is enough and `--durations=0` and `--strict-markers` are not necessary for this package.

---

ok